packages/storybook/src/schematics/storybook-configure/storybook-configure.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  apply,
+  chain,
+  mergeWith,
+  move,
+  Rule,
+  schematic,
+  SchematicContext,
+  Tree,
+  url
+} from '@angular-devkit/schematics';
+import { getWorkspace } from '@schematics/angular/utility/config';
```

Comment:
Let's not depend on `schematics/angular` since this should support React and web components as well.